src/select.jsx

```diff
@@ -344,7 +344,7 @@ var classBase = React.createClass({
           </div>
           : ''
         }
-        <select name={this.props.selectName} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
+        <select name={this.props.selectName} onChange={() => {}} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
```

Comment:
Possibly. Since this <select/> element wasn't using any sort of onChange() function to begin with, I'm not sure what side effects might happen if we did that. It probably wouldn't give us any benefit, either.